checkpoint/types/msg.go

```diff
@@ -168,8 +168,7 @@ func (msg MsgCheckpointAck) ValidateBasic() sdk.Error {
 		return hmCommon.ErrInvalidMsg(hmCommon.DefaultCodespace, "Invalid from %v", msg.From.String())
 	}
 
-	childBlockInterval := helper.GetConfig().ChildBlockInterval
-	if msg.HeaderBlock > 0 && msg.HeaderBlock%childBlockInterval != 0 {
+	if msg.HeaderBlock > 0 {
```

Comment:
It doesn't need `HeadeBlock` check now. Before it was checking if the header block is multiples of block interval. Now we don't need it.